Makefile

```diff
@@ -86,6 +87,9 @@ build: $(TESTS) $(OUT) $(SNIPPET_OUT) $(SEGMENT_SNIPPET_OUT) README.md
 	@$(DUO) --development test/tests.js > build/tests.js
 	@$(DUO) --development test/snippet-tests.js > build/snippet-tests.js
 
+docs:
```

Comment:
I considered adding this to the default make command, but the html files are timestamped, so they would be modified with every make